gammapy/scripts/ts_image.py

```diff
@@ -77,14 +77,18 @@ def ts_image(input_file, psf, model, scales, downsample, residual, morphology,
         maps.append(fits.ImageHDU(data, header, 'OnModel'))
     results = compute_ts_map_multiscale(maps, psf_parameters, scales, downsample,
                                         residual, morphology, width)
-    folder = morphology.lower()
-    if not os.path.exists(folder):
+
+    folder, filename = os.path.split(output_file)
+    if not os.path.exists(folder) and folder != '':
         os.mkdir(folder)
 
     # Write results to file
     header = maps[0].header
-    for scale, result in zip(scales, results):
-        filename = os.path.join(folder, 'ts_{0:.3f}.fits'.format(scale))
-        logging.info('Writing {0}'.format(filename))
-        result.write(filename, header, overwrite=overwrite)
-
+    if len(results) > 1:
+        for scale, result in zip(scales, results):
+            filename_ = filename.replace('.fits', '_{0:.3f}.fits'.format(scale))
+            logging.info('Writing {0}'.format(os.path.join(folder, filename_)))
```

Comment:
No need to put `{0}`, you can just write `{}` because we don't support Python 2.6 and in Python 2.7 and 3 one doesn't have to give the index of the argument to put in `str.format`.